help/scim.md

```diff
@@ -174,10 +174,11 @@ reflected in your SCIM provider, and instead will cause the state of the Zulip
 group to become inconsistent with the state of the SCIM provider's group.
 
 Zulip supports [adding user groups to other
-groups](/help/manage-user-groups#add-user-groups-to-a-group), while some SCIM
-providers (including Okta) do not have this feature. In this case, you can
-combine adding groups to other groups in Zulip with using your SCIM provider to
-manage which individual users are in each group.
+groups](/help/manage-user-groups#add-user-groups-to-a-group), but some SCIM
+providers (including Okta) do not. As a result, this concept is also not
+supported in Zulip's SCIM integration. If you want to use nested groups, you can
+add groups to other groups in Zulip and manage the individual members of each
+subgroup in your SCIM provider.
```

Comment:
reworded this a bit to be clear that nested groups are not supported via SCIM - since I feel the original wording sounded a bit like it was a problem of providers such as Okta, while others might be able to sync group nesting via SCIM (they cannot.)